frontend/common/SocialAuthLogin.tsx

```diff
@@ -46,14 +27,14 @@ export function SocialAuthLogin(props: SocialAuthLoginProps) {
   }
 
   return (
-    <Container>
-      <Heading>{t`Log in with`}</Heading>
-      <Grid>
+    <>
+      <Title headingLevel="h3">{t`Log in with`}</Title>
```

Comment:
From the PF example they do not add this section. Do we need this one?

Reply:
The PF version shows icon where we show buttons with text.

Instead of having buttons like
- Login with Google
- Login with Github
- Login with OIDC

The UXD design is to have one "Login with" and then buttons with simpler labels.
This removes all the repetitive text.